packages/usa-nav/src/styles/_usa-nav.scss

```diff
@@ -342,10 +342,10 @@ $expand-less-icon: map-merge(
       display: inline;
       padding-left: units(0.5);
 
-      & + .usa-nav__secondary-item::before {
-        color: color("base-lighter");
-        content: "|";
-        padding-right: units(0.5);
+      & + .usa-nav__secondary-item {
+        border-left: 1px solid color("base-lighter");
+        margin-left: px-to-rem(5px);
+        padding-left: units(1);
```

Comment:
# Alternative

Alternate solution keeps the pseudo element, but creates the shape. I liked that it's similar to the current, but it's more CSS.

```scss
      & + .usa-nav__secondary-item::before {
        background-color: color("base-lighter");
        display: inline-block;
        width: units("1px");
        height: units(2);
        content: "";
        margin-right: units(0.5);
        vertical-align: middle;
      }
```

Reply:
I noticed that this solution causes some slight content shift brought on by the `height` setting. Here's a video of me clicking back and forth between the development build and local


https://user-images.githubusercontent.com/25211650/216679764-2c1de705-f81b-4187-bd30-41877693a7a2.mov